src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -11,7 +11,7 @@ import { appendToSpreadsheet } from './googleSheetsInternalAPI';
 
 const GOOGLE_SHEETS_DATE_FORMAT = 'DD/MM/YYYY';
 const DEFAULT_SHEET_NAME = '_caspion';
-const COLUMN_HEADERS = ['תאריך', 'סכום', 'תיאור', 'תיאור נוסף', 'קטגוריה', 'מספר חשבון', 'hash - לא לגעת', 'מטבע'];
+const COLUMN_HEADERS = ['תאריך', 'סכום', 'תיאור', 'תיאור נוסף', 'קטגוריה', 'מספר חשבון', 'hash - לא לגעת', 'תאריך חיוב', 'מזהה', 'סטטוס', 'סוג', 'תשלומים', 'סכום מקור', 'מטבע'];
```

Comment:
```suggestion
const COLUMN_HEADERS = [
  'תאריך', 'סכום', 'תיאור', 'תיאור נוסף',
  'קטגוריה', 'מספר חשבון', 'hash - לא לגעת',
  'תאריך חיוב', 'מזהה', 'סטטוס', 'סוג',
  'תשלומים', 'סכום מקור', 'מטבע'
];
```

You need to break the line because it is too long.

Reply:
ok. got it